lxd/storage/drivers/driver_zfs_utils.go

```diff
@@ -82,8 +82,6 @@ func (d *zfs) createDataset(dataset string, options ...string) error {
 }
 
 func (d *zfs) createVolume(dataset string, size int64, options ...string) error {
-	size = d.roundVolumeBlockSizeBytes(size)
```

Comment:
Why did this call get removed?

Reply:
Are all the calls to `createVolume` covered by earlier calls to the rounding function?

Reply:
I wanted to avoid adding a `vol Volume` param to `createVolume`. Alternatively `createVolume` could take a volume and call `ConfigSize` rather than using an additional `size` param, but I thought that expanded the scope of the function a little more than was maybe ideal. Let me know what you think.

Yes, `createVolume` is only used once (`driver_zfs_volumes.go:213`) and I included a call to `roundVolumeBlockSizeBytes`.